ecmascript/transforms/src/compat/es2020/class_properties.rs

```diff
@@ -586,7 +586,10 @@ impl ClassProperties {
                         c.params = c.params.move_map(|mut param| match &mut param {
                             ParamOrTsParamProp::TsParamProp(p) => match &p.param {
                                 TsParamPropParam::Ident(i) => {
-                                    typescript_constructor_properties.push(store(i));
+                                    typescript_constructor_properties.push(store(&Ident {
+                                        type_ann: None,
```

Comment:
I think `type_ann` will be dropped by typescript_strip pass, isn't it?

Reply:
No, seems like `type_ann` has not been dropped

Reply:
Ok, I got it.

Reply:
I got this output:

```js
class BadRequestError extends Error {
    constructor(message){
        super(message);
        this.message: string = message;
    }
}
```

from

```ts
class BadRequestError extends Error {
  constructor(public readonly message: string) {
    super(message)
  }
}
```